spec/post_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'is not valid without a title' do
+    post = Post.new(title: nil)
+    expect(post).to_not be_valid
+  end
+
+  it 'is not valid with a title longer than 250 characters' do
+    post = Post.new(title: 'a' * 251)
+    expect(post).to_not be_valid
+  end
```

Comment:
- [ ] Any custom method in your models should have corresponding tests. For example, the `update_posts_counter` and `recent_comments` methods in your Post model should be tested

Reply:
![image](https://github.com/lerfast/blog-app/assets/111823606/e0d379ed-d976-4963-b990-dece5bc6441c)
done!!!